docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,42 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.
+
+
+## Step 1 : Pre-requisites for accessing the Vendasta's SCIM APIs
+
+### 1. Namespace
+
+This must be the id of the partner that wish to manage users in sync with Vendasta. It should be unique for a partner.
+
+### 2. Authorization token
+- For a partner with Vendasta, in order to access APIs of Vendasta they must need a authorisation token which is generated against the namespace of the partner. This should be refreshed over a particular time.
+
+- The authorization token must be generated with required scopes in order to access a particular platform.
+
+### 3. Scope
+- It is used to specify the access rights the partner should have towards Vendasta’s SCIM APIs.
```

Comment:
**user.admin** is the only scope that we are going to handle for the SCIM API's. So, we don't need to mention it as a Pre-requisite item.

We can add scope details in the Authorization token (point 2) by specifying that each token must have a user.admin scope